content/code-of-conduct.md

```diff
@@ -10,7 +10,7 @@ do not tolerate harassment of participants in any form.
 This code of conduct applies to all STL Tech spaces, including the STL Tech
 Slack and Github Repositories, both online and off. Anyone who violates this
 code of conduct may be sanctioned or expelled from these spaces at the
-discretion of the the incident response team.
+discretion of the the incident response team. **Please note that STL Tech spaces are not public spaces** and that interpretation of this Code of Conduct in matters of sanctioning or expelling participants in violation of same is at the sole discretion of the incident response team.
```

Comment:
Should the note for this be on the same line, or a new line?

Reply:
oh good point, might be better on a new line